packages/kbn-grid-layout/grid/test_utils/sample_layout.ts

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { GridLayoutData } from '../types';
+
+export const getSampleLayout = (): GridLayoutData => [
```

Comment:
initially I was wondering if we should test this from the grid_example app, but noticed we don't really add tests there... I am happy to move it there if it makes sense, as this layout object is copied from there.

Reply:
This is fine - I'm making significant changes to the layout object used in the example plugin in my embeddable integration PR (https://github.com/elastic/kibana/pull/198413) so this will no longer be duplicated at that point :) I think the tests belong in the `kbn-grid-layout` package